<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMU_Calibration</class>
 <widget class="QMainWindow" name="FMU_Calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>19</y>
      <width>731</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_pump">
        <attribute name="title">
         <string>Pump Calibration</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>19</y>
           <width>701</width>
           <height>431</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Please do all three calibration</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Warning: The machine will dispense during calibration.
  Press Calibrate, measure dispensed milk, fill out volume in ml, press set</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" columnstretch="2,0,0,0,0">
            <item row="2" column="2">
             <widget class="QProgressBar" name="pb_3">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="btn_cal_50">
              <property name="text">
               <string>cailbration</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>pump calibration speed 80%</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QProgressBar" name="pb_1">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="btn_cal_spd_2">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="btn_cal_100">
              <property name="text">
               <string>cailbration</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>pump calibration speed 100%</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="btn_cal_spd_3">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QDoubleSpinBox" name="sp_cal_spd_4">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>150.000000000000000</double>
              </property>
              <property name="value">
               <double>103.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QProgressBar" name="pb_2">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="btn_cal_spd_1">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>pump calibration speed 30%</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="btn_cal_80">
              <property name="text">
               <string>cailbration</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btn_cal_30">
              <property name="text">
               <string>cailbration</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>pump calibration speed 50%</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="4">
             <widget class="QPushButton" name="btn_cal_spd_4">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QProgressBar" name="pb_4">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QDoubleSpinBox" name="sp_cal_spd_1">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="value">
               <double>35.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="sp_cal_spd_2">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="value">
               <double>62.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QDoubleSpinBox" name="sp_cal_spd_3">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="value">
               <double>87.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
